skins/angeldao.scss

```
@@ -0,0 +1,14 @@
+.benchmark {
```

The css selector should similar to the skin file name. `skin.scss` `.skin {}`.
```suggestion
.angeldao {
```